### ISSUE TITLE
Context Menu Visual Issue with Windows Terminal

### ISSUE THREAD

**Issue by KrisSchneider** [moudey/shell#15]
Hello @moudey, I hope your doing well.

Can you help me fix this visual issue with Windows Terminal?
![(brave)z4r0P1_06-08-2022](https://user-images.githubusercontent.com/101912712/172757018-bbe4ed60-00ce-4d45-b5d2-05124d3874a9.jpg)

The shortcut for Windows Terminal in the context menu is now displaying an extremely long tag for it.

Here are my files if you want more context.
[Nilesoft Stuff.zip](https://github.com/moudey/shell/files/8866526/Nilesoft.Stuff.zip)

**Comment by moudey** [comment]
Hello @KrisSchneider 

You may need permissions to access the Windows Terminal path. Anyway, try [this](https://github.com/moudey/shell/files/8874562/shell.zip) where the package name was fetched from the MrtCache registry

**Comment by KrisSchneider** [comment]
Hello @moudey, 

I managed to find the Mrtcache in the registry like you asked. I have no idea what to do with the .dll file you gave me though.
![(regedit)71J780_06-09-2022](https://user-images.githubusercontent.com/101912712/172987200-70cadd4e-1a08-4db0-88b2-19126ce6acfb.jpg)

Now what I am supposed to do now?

**Comment by moudey** [comment]
Unregister Shell and replace the attached file with shell.dll, then register Shell and tell me the result

**Comment by KrisSchneider** [comment]
Hello @moudey, sorry for being late. I was busy finishing college assignments. Anyway, I did try your method. I opened shell.exe with admin privileges. Clicked “unregister” and restarted File Explorer. Then in that folder where I placed Nilesoft at, I replaced the old 'shell.dll' file with YOUR 'shell.dll'. Afterwards I opened shell.exe with admin privileges again, clicked “register” and restarted File Explorer. 

It did fix my issue, now the context menu is back to normal again!
![(explorer)45l722_06-11-2022](https://user-images.githubusercontent.com/101912712/173178053-146863c2-074c-44dc-acb4-c9d48c4b9649.jpg)
